docs/tutorial/dapr-ambient-and-knative.md

```diff
@@ -0,0 +1,201 @@
+  name: kourier-ingress
+  namespace: kourier-system
+  labels:
+    networking.knative.dev/ingress-provider: kourier
+spec:
+  type: NodePort
+  selector:
+    app: 3scale-kourier-gateway
+  ports:
+    - name: http2
+      nodePort: 31080
+      port: 80
+      targetPort: 8080
+EOF
+```
+
+
+Let's create a Redis instance that our applications can use to store state or exchange messages: 
+
+```
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update                            
+helm install redis bitnami/redis --set image.tag=6.2 --set architecture=standalone
+```
+
+Finally, let's install Dapr into the Cluster: 
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+helm upgrade --install dapr dapr/dapr \
+--version=1.10.4 \
+--namespace dapr-system \
+--create-namespace \
+--wait
+```
+
+Let's now deploy and configure some Dapr apps! 
+
+
+## Deploying the applications and wiring things together
+
+In this section, we will be deploying three applications that want to store and read data from a state store and publish and consume messages. 
+To achieve this we will use the Dapr StateStore and PubSub components. So before deploying our applications let's configure these components to connect the Redis instance that we created before. 
+
+The Dapr Statestore configuration looks like this: 
+```
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.redis
+  version: v1
+  metadata:
+  - name: keyPrefix
+    value: name
+  - name: redisHost
+    value: redis-master:6379
+  - name: redisPassword
+    secretKeyRef:
+      name: redis
+      key: redis-password
+auth:
+  secretStore: kubernetes
+```
+
+We can apply this resource to Kubernetes by running: 
+```
+kubectl apply -f resources/statestore.yaml
+```
+
+The PubSub Component looks like this: 
+```
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: notifications-pubsub
+spec:
+  type: pubsub.redis
+  version: v1
+  metadata:
+  - name: redisHost
+    value: redis-master:6379
+  - name: redisPassword
+    secretKeyRef:
+      name: redis
+      key: redis-password
+auth:
+  secretStore: kubernetes
+```
+
+We can apply this resource to Kubernetes by running: 
+```
+kubectl apply -f resources/pubsub.yaml
+```
+
+Once we have the PubSub component configured, we can register Subscritions to define who and where notifications will be sent when new messages arrive to a certain topic. A Subscription resource look like this: 
+
+```
+apiVersion: dapr.io/v1alpha1
+kind: Subscription
+metadata:
+  name: notifications-subscritpion
+spec:
+  topic: notifications 
+  route: /notifications
+  pubsubname: notifications-pubsub
+```
+
+Finally, let's deploy three applications that uses the Dapr StateStore and PubSub components. This are normal/regular Kubernetes applications, using Deployments and Services. To make these apps dapr-aware we just need to add some Dapr annotations:
+
+
+```
+
+```
+
+Let's deploy the apps with: 
```

Comment:
@mcruzdev These applications can be found here: https://github.com/salaboy/dapr-example-apps 
But you need to replace how they connect to the ambient instance: 
Doing something like this: 
```
var daprClient dapr.Client
var DAPR_HOST = "my-ambient-dapr-ambient.default.svc.cluster.local"
var DAPR_PORT = "50001"

type MyValues struct {
	Values []string
}


	daprClient, err := dapr.NewClientWithAddress(net.JoinHostPort(DAPR_HOST, DAPR_PORT))
	if err != nil {
		panic(err)
	}

```

When you create a Dapr client, you need to point it to your Dapr Ambient instance.